File: GO.ejercicios/interaccion.usuario.go
package main

import (
	"fmt"
	"strings"
)

func main() {
	// Variables para almacenar los datos del usuario
	var nombres, apellidos, profesion string
	var anioNacimiento int

	// Pedir al usuario que ingrese sus datos
	fmt.Print("Escriba sus nombres completos: ")
	fmt.Scanln(&nombres)

	fmt.Print("Escriba sus Apellidos completos: ")
	fmt.Scanln(&apellidos)

	fmt.Print("Escriba su profesion: ")
	fmt.Scanln(&profesion)

	fmt.Print("Escriba su año de nacimiento: ")
	fmt.Scanln(&anioNacimiento)

	// Calcular la edad en base al año actual (2025)
	edad := 2025 - anioNacimiento

	// Mostrar el resultado
	fmt.Printf("El (La) %s %s %s tiene %d años\n", profesion, nombres, apellidos, edad)
}

File: RUBY.EJERCICIOS/interaccion.usuario.rb
# Solicitar al usuario sus datos personales

# Pedir al usuario que ingrese sus datos
print "Escriba sus nombres completos: "
a = gets.chomp  # Lee el nombre completo

print "Escriba sus Apellidos completos: "
b = gets.chomp  # Lee los apellidos completos

print "Escriba su profesion: "
c = gets.chomp  # Lee la profesión

print "Escriba su año de nacimiento: "
d = gets.to_i  # Lee el año de nacimiento

# Calcular la edad en base al año actual (2025)
e = 2025 - d

# Mostrar el resultado
puts "El (La) #{c} #{a} #{b} tiene #{e} años"